llap-client/src/java/org/apache/hadoop/hive/llap/registry/impl/LlapZookeeperRegistryImpl.java

```diff
@@ -168,6 +168,10 @@ public String register() throws IOException {
     String uniqueId = UNIQUE_ID.toString();
     long znodeCreationTimeout = 120;
 
+    // PersistentNode created on server will be preserved on VW restarts which will make
```

Comment:
"VW restart" is a Cloudera hive ware warehouse specific thing, let's try to explain this in another way which describes our scenario

Reply:
Just this comment is not proper for upstream